connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -70,20 +70,30 @@ public class WorkerConfig extends AbstractConfig {
     public static final String CLIENT_DNS_LOOKUP_CONFIG = CommonClientConfigs.CLIENT_DNS_LOOKUP_CONFIG;
     public static final String CLIENT_DNS_LOOKUP_DOC = CommonClientConfigs.CLIENT_DNS_LOOKUP_DOC;
 
+    public static final String PLUGIN_VERSION_SUFFIX = "version";
```

Comment:
I checked the KIP, and the suffix should be `plugin.version`.

But the connector is still written to use "version", is that a mistake or intentional? I would prefer it being consistent so we can use the same constant everywhere.

Reply:
This should be `plugin.verison`.

I changed it from `version` to `plugin.version` based on a suggestion in the KIP. I only did it for converters and transformations plugins since in those the `value.converter` prefix and `version` is passed to validate the connector config, which can conflict if the converter config def defined it internally. But `connector.version` is more specific and internal connector config def should not ideally define it. That said, making it consistent across connector and converters is good too. Will make this change.